jupyterhub/templates/image-puller/_daemonset-helper.yaml

```diff
@@ -83,7 +80,6 @@ spec:
         {{- range $k, $v := .Values.prePuller.extraImages }}
         - name: image-pull-{{ $k }}
           image: {{ $v.name }}:{{ $v.tag }}
-          imagePullPolicy: {{ $v.policy | default "IfNotPresent" }}
```

Comment:
This is deleting an option from the configuration. Maybe this should be conditional `if $v.policy` and only set it if defined?

Reply:
Yes in the case of extraImages that would make sense. But we must not provide a truthy default value in `values.yaml` if we create such conditionals - and we won't for `extraImages`.

Perhaps we should remove the default pullPolicies in `values.yaml` and create conditionals throughout of declaring the imagePullPolicy, as the omission is an option in itself.

Reply:
To formulate things easier:

Do you agree that defining `"IfNotPresent"` as default value in `values.yaml` is bad, as the default of omission is what's really what is expected?

Reply:
@consideRatio yeah, if it is set in config, we should set it. If not, we should omit it.